org.eclipse.winery.bpmn4tosca.converter.tobpel/src/main/java/org/eclipse/winery/bpmn2bpel/model/GatewayBranch.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright 2017 ZTE Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ * Contributors:
+ *     ZTE - support of more gateways
+ *******************************************************************************/
+package org.eclipse.winery.bpmn2bpel.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class represents a sub path of a gateway node
+ * 
+ * it contains two properties, id is the first following node's id; 
+ * nodeList is the nodes of the branch order by flow sequence.
+ *
+ */
+public class GatewayBranch {
+	
+	protected String id;
```

Comment:
sorry, i'm not sure what does 'ident did go wrong' mean. Is this about the variable modifier? do you think this modifier should be private?